skyportal/handlers/api/group.py

```diff
@@ -80,7 +81,9 @@ def get(self, group_id=None):
                                 type: array
                                 items:
                                   $ref: '#/components/schemas/Group'
-                                description: List of all groups if current user is Super admin, else None.
+                                description: |
+                                  List of all groups user has access to, optionally
+                                  including single user groups if specified in request.
```

Comment:
What does it mean to "specify in request"?

Reply:
e.g. the parameter `includeSingleUserGroups` described above equals `true`

Reply:
Updated docs